### ISSUE TITLE
Rance VI Mangagamer 2.05A - Screen Select Thumbnails Wrong

### ISSUE THREAD

**Issue by ripdog** [nunuhara/xsystem4#115]
Hi there, thanks for the hard work on this program!

I've recently started playing the Mangagamer version of Rance VI, with the English version of 2.05A patch applied on xsystem4 alpha-4.1 on Win11, using the DirectX renderer.

When I'm in the Base Screen and click Scene Select, I see this:

![image](https://user-images.githubusercontent.com/174944/235416183-ec2c1722-5d73-48b9-8514-0feb3d9c824f.png)

Mostly correct, but the Base Screen thumbnail is incorrect. However, if I go into Level Goddess and then back to Scene Select...:

![image](https://user-images.githubusercontent.com/174944/235416254-d5b9bdf8-014f-4b0f-8904-0b24a0b32809.png)

There are now three incorrect thumbnails.

---

While I have you here, is it possible to add a feature to slow down CTRL-skipping? Currently, the skipping is so fast events often finish before I can release the button, even which I try and only tap the button quickly. Gimme the word and I'll split this into a new issue.

**Comment by ripdog** [comment]
Just realised the game leaves a console window open. These warnings appear when opening screen select:

```
*WARNING*(../src/hll/SACT2.c:SACT2_WP_GetSP:137): Unimplemented HLL function: SACT2.WP_GetSP
*WARNING*(../src/video.c:gfx_save_texture:590): Failed to open ..//./Rance6/SaveData/ScreenA.qnt: No such file or directory
*WARNING*(../src/hll/SACT2.c:SACT2_WP_GetSP:137): Unimplemented HLL function: SACT2.WP_GetSP
*WARNING*(../src/video.c:gfx_save_texture:590): Failed to open ..//./Rance6/SaveData/ScreenB.qnt: No such file or directory
*WARNING*(../src/hll/SACT2.c:SACT2_WP_GetSP:137): Unimplemented HLL function: SACT2.WP_GetSP
*WARNING*(../src/video.c:gfx_save_texture:590): Failed to open ..//./Rance6/SaveData/ScreenD.qnt: No such file or directory
*WARNING*(../src/hll/File.c:File_GetTime:165): stat failed: No such file or directory
*WARNING*(../src/hll/File.c:File_GetTime:165): stat failed: No such file or directory
*WARNING*(../src/hll/File.c:File_GetTime:165): stat failed: No such file or directory
*WARNING*(../src/hll/File.c:File_GetTime:165): stat failed: No such file or directory
*WARNING*(../src/hll/File.c:File_GetTime:165): stat failed: No such file or directory
*WARNING*(../src/hll/File.c:File_GetTime:165): stat failed: No such file or directory
*WARNING*(../src/hll/File.c:File_GetTime:165): stat failed: No such file or directory
*WARNING*(../src/hll/SACT2.c:SACT2_WP_GetSP:137): Unimplemented HLL function: SACT2.WP_GetSP
*WARNING*(../src/video.c:gfx_save_texture:590): Failed to open ..//./Rance6/SaveData/ScreenE.qnt: No such file or directory
*WARNING*(../src/video.c:gfx_save_texture:590): Failed to open ..//./Rance6/SaveData/ScreenF.qnt: No such file or directory
```

**Comment by nunuhara** [comment]
Thanks for the report. This should be fixed in [alpha-4.2](https://github.com/nunuhara/xsystem4/releases/tag/alpha-4.2).
Please open a separate issue for CTRL-skipping. That one will be tricker to address.

**Comment by ripdog** [comment]
Can confirm it's fixed! Thank you very much.
